build.gradle

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
No need to change the license year for edited files.

Reply:
Sure, Changed with [b20f978](https://github.com/ballerina-platform/edi-tools/pull/28/commits/b20f97803a9c3b57c7e0e33ad3d4cc088062593c).